docs/proposals/tenant_retention.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "Retention of Tenant Data from Blocks Storage"
+linkTitle: "Retention of Tenant Data from Blocks Storage"
+weight: 1
+slug: tenant-retention
+---
+
+- Author: [Allenzhli](https://github.com/Allenzhli)
+- Date: December 2020
+- Status: Proposed
+
+## Retention of tenant data
+
+## Problem
+
+In Cortex, we are missing retention of blocks storage. The metric data is growing over time per-tenant, at the same time, the value of data decreases. We want to support retention time of metric data like prometheus does.
```

Comment:
This is not entire correct. Data retention is typically achieved be achieved via a bucket policy. However, we should explain this has two main issues:

1. Not every backend storage support bucket policies
2. Bucket policies don't easily allow a per-tenant custom retention

Reply:
I agree with you. I will update this.